### PR TITLE
feat: change message and handle connect function

### PR DESCRIPTION
**Pull request by J4ckyLIM**
- Fix typo
- Change modal message during signMessage
- Add toastpromise during connect function

Preview:

https://user-images.githubusercontent.com/42095148/195162376-a7ec3783-84f3-481a-99cd-12c3262521c2.mov

### CODE REVIEW

**Comment by lechinoix** [comment]
Hey @J4ckyLIM, thanks for the Pull Request, seems like good UX improvements ! Would you have screenshots of the result on your computer ? I'll let @ltoussaint or @Bernardstanislas review this tomorrow :)

**Review by Bernardstanislas — APPROVED** [review]
Looks good, I'd love a visual preview too :)

**Comment by Bernardstanislas** [comment]
Will be able to preview it on the Vercel app actually 🤯

File: src/App/Routes/Modals/RegisterModal/GithubSignin/index.tsx [hunk 1]
```diff
@@ -40,12 +41,31 @@ const GithubSigninContainer: FC<PropsWithChildren<Props>> = ({ children, classNa
...
+          address: account.address,
+          code,
+          hash,
+          signature: signature as [string, string],
+        }),
+        {
+          success: () => {
+            return (
+              <div className="leading-[1rem] line-clamp-3">
+                Congratulation, your account have been successfully connected !
```
Comment (Bernardstanislas) [inline src/App/Routes/Modals/RegisterModal/GithubSignin/index.tsx:55]:
```diff
- Congratulation, your account have been successfully connected !
+ Congratulations, your account has been successfully connected!
```

**Comment by ltoussaint** [comment]
> Will be able to preview it on the Vercel app actually 🤯 

Not so sure, as it requires GitHub connection, you will not be able to perform it  on preview environment

**Pull request merged**